cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/PredicateFunctionsTest.scala

```diff
@@ -98,13 +98,13 @@ class PredicateFunctionsTest extends DocumentingTest {
       }
     }
     section("exists()", "functions-exists") {
-      p("""The function `exists()` returns `true` if a match for the given pattern exists in the graph, or if the specified property exists in the node, relationship or map.
+      p("""The function `exists()` returns `true` if a match for the given pattern exists in the graph.
           #`null` is returned if the input argument is `null`.""".stripMargin('#'))
```

Comment:
I think that perhaps `exists` should now really be either an expression or a clause. In the code I've seen it called ExistsSubqueryClause although it extends Expression.

The other point I would like to raise is that exists() is not valid syntax for existential subqueries. It needs curly braces instead, eg exists { }.

Reply:
Existential subqueries is documented in another place. This is now only for `exists(pattern)`, which I agree is not really a function even if it happens to be implemented as one. I will ask someone from docs team to review this and then they can maybe come with a concrete suggestion on how to rephrase/move this part of documentation to make it clearer

Reply:
I agree that it may be confusing with both the function `exists()` and the existential subquery `EXISTS {}`, but whether this should or shouldn't be is beyond me. If it is implemented as a function, it belongs in this section. 
If you feel the risk of confusion between the two is too great, you could put a note about it in the Predicate functions section, not in the glossary.